lib/rules/comment-pattern/README.md

```diff
@@ -0,0 +1,39 @@
+# custom-property-pattern
+
+Specify a pattern for comments.
+
+<!-- prettier-ignore -->
+```css
+                  /*comment*/
+                 /* ↑ */ a { --foo-: 1px; }
+/**                 ↑
+ * The pattern of this */
+```
+
+## Options
+
+`regex`
+
+The pattern to match the comments with.
+
+Given the regex:
+
+```js
+/foo .+/
+```
```

Comment:
I would use [this](https://github.com/stylelint/stylelint/blame/924646681a355b327ccfeb6f00be4e1636333911/lib/rules/custom-property-pattern/README.md#L14-L22) and only adjusted RegEx.

If rule accept RegEx, then we allow it to be both string and RegEx, because JSON and other formats doesn't have RegEx as JavaScript does.